pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -203,7 +203,7 @@ func (pl *PodTopologySpread) calPreFilterState(pod *v1.Pod) (*preFilterState, er
 	if len(pod.Spec.TopologySpreadConstraints) > 0 {
 		// We have feature gating in APIServer to strip the spec
 		// so don't need to re-check feature gate, just check length of Constraints.
-		constraints, err = filterTopologySpreadConstraints(pod.Spec.TopologySpreadConstraints, v1.DoNotSchedule)
+		constraints, err = filterTopologySpreadConstraints(pod.Spec.TopologySpreadConstraints, v1.DoNotSchedule, pl.enableMinDomainsInPodTopologySpread)
```

Comment:
Where is the code that uses the MinDomains?

Reply:
I was thinking of implementing the core logic of MinDomains in the next PR, but should I include it in this PR?

Reply:
Usually we either split only API changes and adding the feature gate in one PR and another one for using the field. Or you might just put everything together, if the changes are small enough.

But since this is already reviewed, it's fine. This can't merge until the implementation is ready though. Better do it in a separate PR at this point.

Reply:
Okay. I'll prepare the PR for the implementation.

Reply:
Open #108362